### ISSUE TITLE
Support multiple accounts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#87]
```
I've searched about it but I couldn't find nothing related
so here I am :

I -and I believe many others- would love to have more that one twitter 
accounts in Pino. It may be a preference to switch to another account, 
anytime.

I do not know the way and how hard it would be to implement.
I'm just expressing my proposal.

I've seen it and so I'm asking for it 'cuase it will be helpful imo.

Btw, great update today (0.1.5)


Ioannis,
```

Original issue reported on code.google.com by `iovarsa...@gmail.com` on 11 Feb 2010 at 7:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 11 Feb 2010 at 7:39
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 31f0130222 revision.
```

Original comment by `tro...@gmail.com` on 17 Feb 2010 at 7:35
- Changed state: **Fixed**
